.github/scripts/random1on1pairings

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import argparse
```

Comment:
I have never heard of absl-- I'll take a look into it

Reply:
Just looked into `absl`

I don't understand what would be missing from `argparse` that would require that we use `absl`.

I'm inclined to say we shouldn't adopt it just because `argparse` is core python and does everything we need (_as far as I know_)